com.microsoft.mrtk.core/Editor/MRTKProjectValidation.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.XR.CoreUtils.Editor;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// A class helping users with validating the configuration of the project for use with MRTK3
+    /// </summary>
+    public static class MRTKProjectValidation
+    {
+        private const string XRProjectValidationSettingsPath = "Project/XR Plug-in Management/Project Validation";
+        private const string DefaultMRTKProfilePath = "Packages/com.microsoft.mrtk.core/Configuration/Default Profiles/MRTKProfile.asset";
```

Comment:
Can we load this by GUID or something instead? Just in case we ever change this path and forget to update it here...

Reply:
Fixed, thanks!